crates/rpc/tests/rpc.rs

```diff
@@ -23,11 +23,13 @@ lazy_static::lazy_static! {
     static ref BLOCK_CACHE: Mutex<SizedCache<u64, MaybePendingBlockWithTxs>> = Mutex::new(SizedCache::with_size(1000));
 }
 
+fn rpc_url() -> Url {
+    let config = Config::from_env();
```

Comment:
(follow up on https://github.com/eigerco/beerus/pull/631 && https://github.com/eigerco/beerus/pull/631#issuecomment-2066467473)

> I really wish is was possible to fix one little thing without having to fix everything that's around, significantly extending the scope of an initially trivial PR, and bringing multiple breaking changes in the process. I'm closing this PR.
> 
> #630 will use config files.

---

_(Just trying to understand the bigger picture behind this PR and #630 by asking some questions. They are not suggestions, just questions, no need to act on them. But answers to those questions I believe might be useful to choose the rational way to proceed here.)_

Why require to parse the full config here if only RPC URL is necessary? Won't it be easier to just lookup necessary env var for the RPC URL or am I missing something?

With env var lookup even the extraction of the `rpc_url` function is not necessary at this point, as it is only being used once (unless I am missing something here too).

With such env var simplification I fail to see the necessity of https://github.com/eigerco/beerus/pull/631 as a prerequisite for this PR. Please enlighten me if I am indeed missing the bigger picture here.

Reply:
This env var doesn't exist, that's why I initially looked into #631.

Now these workflows could run sequentially, but it makes the overall assessment more tedious: I can only have them run about 12 times in a day and the reliability assessment has less value.
If I run them in parallel, I would either need 3 self-hosted runners, or different ports, or would quickly run out of GitHub actions credits.

Reply:
Hm, it looks like I would need 3 runners anyway, making the port irrelevant.

Reply:
I removed these changes as they are now obsolete.